### ISSUE TITLE
Could not register Hotkey: Hotkey support broken on Python 3.10

### ISSUE THREAD

**Issue by xeruf** [hhannine/superpaper#102]
**Describe the bug**

Using KDE Plasma, I keep getting popups with such an error whenever I start Superpaper or try to update the wallpaper:

    Error: could not register hotkey ('control', 'super', 'shift', 'h'). Check that it is formatted properly and valid keys.

**Desktop (please complete the following information):**
 - OS: Arch
 - Desktop Environment on Linux: KDE
 - Superpaper package & version: AUR 2.1.0

**Comment by hhannine** [comment]
Are you running Python 3.10? It could be that your issue is rooted in a breakage of the hotkey library that I've now noticed with Python 3.10.

If that's not the case, I think I need to see a debug log from you.

**Comment by xeruf** [comment]
I am on Arch, so I am running whatever the latest version is :)
```sh
❯ python --version
Python 3.10.2
```

**Comment by hhannine** [comment]
Thanks for verifying the version. I've found a solution but it requires the modification of the system_hotkey dependency library. I've discussed the solution in issue #106 . 

Since the maintainer has not yet updated the library, we can fix this ourselves by:
1. clone https://github.com/timeyyy/system_hotkey
2. implement the two code changes of the commit: https://github.com/timeyyy/system_hotkey/pull/26/commits/07799cc6397a992dde353a7ad7cb9e089e7ed4de
3. install the modified system_hotkey by running in the root folder of  the repo: `pip install . --user`
